scripts/configure_pyproject_toml.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
Should we make this an executable Python script for the `./bin` directory?

Reply:
Do you think this will play nicely with Mac and Windows?

Reply:
Support does exist https://peps.python.org/pep-0397/#shebang-line-parsing and  https://docs.python.org/3/using/windows.html#arguments-in-shebang-lines but I'm fine for using as is for now.